rs/sns/governance/CHANGELOG.md

```diff
@@ -53,24 +53,24 @@ to the newest version blessed by the NNS. To do so, please submit a `ManageNervo
 proposal, e.g.:
 
     ```bash
-    dfx canister --network ic call $SNS_GOVERNANCE_CANISTER_ID manage_neuron '(
-    record {
-        subaccount = blob "'${PROPOSER_SNS_NEURON_SUBACCOUNT}'";
-        command = opt variant {
-        MakeProposal = record {
-            url = "https://forum.dfinity.org/t/proposal-opt-in-mechanism-for-automatic-sns-target-version-advancement/39874";
-            title = "Opt for automatic advancement of SNS target versions";
-            action = opt variant {
-            ManageNervousSystemParameters = record {
-                automatically_advance_target_version = opt true;
-            }
+    dfx canister --ic call ${SNS_GOVERNANCE_CANISTER_ID} manage_neuron '(
```

Comment:
This just makes the change log look nicer, as in the original proposal that released these changes (we had these change logs included twice by accident).